sys/linux/vmw_vmci.txt.const

```diff
@@ -0,0 +1,46 @@
+# Code generated by syz-sysgen. DO NOT EDIT.
+arches = 386, amd64, arm, arm64, riscv64, s390x
+AT_FDCWD = ???
```

Comment:
This is defined by some common header file, you can find it in lots of other description files. Please include it here as well.

Reply:
Done